gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/application/OAuthClientSettings.java

```diff
@@ -53,75 +58,6 @@ public class OAuthClientSettings {
     @JsonProperty("renew_client_secret_supported")
     private boolean renewClientSecretSupported;
 
-    public List<String> getRedirectUris() {
-        return redirectUris;
-    }
-
-    public void setRedirectUris(List<String> redirectUris) {
-        this.redirectUris = redirectUris;
-    }
-
-    public String getClientUri() {
-        return clientUri;
-    }
-
-    public void setClientUri(String clientUri) {
-        this.clientUri = clientUri;
-    }
-
-    public String getLogoUri() {
-        return logoUri;
-    }
-
-    public void setLogoUri(String logoUri) {
-        this.logoUri = logoUri;
-    }
-
-    public List<String> getResponseTypes() {
-        return responseTypes;
-    }
-
-    public void setResponseTypes(List<String> responseTypes) {
-        this.responseTypes = responseTypes;
-    }
-
-    public List<String> getGrantTypes() {
-        return grantTypes;
-    }
-
-    public void setGrantTypes(List<String> grantTypes) {
-        this.grantTypes = grantTypes;
-    }
-
-    public String getClientId() {
-        return clientId;
-    }
-
-    public void setClientId(String clientId) {
-        this.clientId = clientId;
-    }
-
-    public String getClientSecret() {
-        return clientSecret;
-    }
-
-    public void setClientSecret(String clientSecret) {
-        this.clientSecret = clientSecret;
-    }
-
-    public String getApplicationType() {
-        return applicationType;
-    }
-
-    public void setApplicationType(String applicationType) {
-        this.applicationType = applicationType;
-    }
-
-    public boolean isRenewClientSecretSupported() {
-        return renewClientSecretSupported;
-    }
-
-    public void setRenewClientSecretSupported(boolean renewClientSecretSupported) {
-        this.renewClientSecretSupported = renewClientSecretSupported;
-    }
+    @JsonProperty("additional_client_metadata")
+    private JsonNode additionalClientMetadata;
```

Comment:
I am not sure to understand why you need a JsonNode here ? Why not directly a map ?